src/VisualStudio/Core/Def/Implementation/SemanticClassificationCache/SemanticClassificationCacheIncrementalAnalyzerProvider.cs

```diff
@@ -40,6 +40,13 @@ private class SemanticClassificationCacheIncrementalAnalyzer : IncrementalAnalyz
         {
             public override async Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, InvocationReasons reasons, CancellationToken cancellationToken)
             {
+                // only process C# and VB.  OOP does not contain files for other languages.
```

Comment:
💭 Seems like it _should_ contain those files

Reply:
yup.  i've opened https://github.com/dotnet/roslyn/issues/47341 to track that.